README.md

```diff
@@ -23,21 +23,73 @@
+
+Jujutsu combines many distinct design features from other version control
+systems into a single user interface. Those other systems include:
+
+- **Git**: Like Git, we make an effort to [be fast][perf], from a snappy
+  UX to efficient algorithms. It also by default shares Git's data model, for
+  wide interoperability and ease of onboarding.
+
+- **Mercurial & Sapling**: Many Mercurial inspired features include the [revset]
+  language to select commits. There is [no explicit index][no-index] or "staging
+  area". Branches are "anonymous" like Mercurial, so you don't need to make up
+  a name for each small change. Primitives for rewriting history are powerful
+  and simple.
+
+- **Pijul & Darcs**: Jujutsu keeps track of conflicts as [first-class objects][conflicts]
+  in its model; it tracks them as if it tracked any other file. While not as
+  rigorous as patch-based systems like Pijul and Darcs, the effect is that many
+  forms of conflict resolution can be performed and propagated automatically.
+
+[perf]: https://github.com/martinvonz/jj/discussions/49
+[revset]: https://martinvonz.github.io/jj/latest/revsets/
+[no-index]: https://martinvonz.github.io/jj/latest/git-comparison/#the-index
+[conflicts]: https://martinvonz.github.io/jj/latest/conflicts/
+
+And it adds several innovative features of its own that those don't have:
+
+- **Working-copy-as-a-commit**: Changes to files are [recorded automatically][wcc]
+  as normal commits, and amended on every subsequent change. This "snapshot"
+  design simplifies the user-facing data model, simplifies internal algorithms,
+  and completely subsumes features like the Git's stash and index/staging-area.
+
+- **Operation log, undo**: Jujutsu records every operation that is performed on the
+  repository, from commits, to pulls, to pushes. This makes debugging problems like
+  "what just happened?" or "how did I end up here?" easier, *especially* when you're
+  helping your coworker answer those questions! And because everything is recorded,
+  you can undo that mistake you just made with ease. Finally: version control
+  has entered [the 1960s][undo-history]!
+
+- **Safe, concurrent replication**: Have you ever wanted to store your version
+  controlled repositories inside a Dropbox folder? No? Well now you can!
+  Fundamentally the problem with using systems like Dropbox for Git or
+  Mercurial, or backing them up with `rsync`, is that that they rely on *local
+  filesystem operations* being atomic, serialized, and non-concurrent with
+  respect to other writes—which is _not_ true when operating on distributed
+  file systems or when operations (like copying files to S3) happen while
+  lock files are being held. Jujutsu is instead designed to be [safe under
+  concurrent][conc-safety] scenarios; rsync and dropbox will never *corrupt*
+  your repository. The worst that will happen is that it will expose conflicts
+  between the local and remote state, leaving you to resolve them.
```

Comment:
This (and some of the above) seems a bit redundant with the "Features" section below. Did you review that in the context of this new text?

Reply:
I was honestly thinking the next course of action after this PR is to move most of the images and follow up description out of the README and onto the website. I think of the README as mostly a "landing page" right now for most people, since obviously its a VCS and they're on GitHub. The website is where we should link users for all examples and descriptions that are more thorough, which I think includes the follow up.

I'd probably put it all on the front of the website. Would that be good?

Reply:
> I was honestly thinking the next course of action after this PR is to move most of the images and follow up description out of the README and onto the website

I think of doing this as a headache because of versioning. I think that's probably solvable with Github actions, but not easily. For example, the scripts I made to generate the screenshots are brittle; they are at least sensitive to what fonts and SVG libraries are installed on a system.

I think it's an OK goal if you want to make it come to pass, but either it will involve doing manual work on each release or it will take some doing. I also think the current setup is good enough for now, so there's no urgency to achieve this goal.

Reply:
I honestly wonder if it would be easier to just do something like an asciinema cast then... I'll look into this a little more, but it can go into a follow up change.

Reply:
Feel free to look into it. Some inconclusive thoughts:

- I am not a fan of the *animation* part of Asciicast. If we do make the demos into animations, it's a different medium, so the demos might need to be adjusted so that important information isn't gone from the screen by the time the story gets to the point where it's needed.
- There are Asciinema-related tools I looked into for generating the screenshots (e.g. https://github.com/asciinema/agg), but I didn't like them as much since, again, they focus on animations. Still, perhaps you'll be more successful in getting them to generate a nice png than I was.